src/app/services/turnstile/turnstile.middleware.ts

```diff
@@ -3,5 +3,6 @@ import { celebrate, Joi, Segments } from 'celebrate'
 export const validateTurnstileParams = celebrate({
   [Segments.QUERY]: Joi.object({
     captchaResponse: Joi.string().allow(null).required(),
+    captchaType: Joi.string().allow('').required(),
```

Comment:
Might have some validation issue where users who were loading older FE (that doesn't supply `captchaType`) hitting the latest BE (that requires `captchaType`). The fix for FE is to refresh the page (which may require them to lose all working data)

Reply:
Very good catch, I think I will have to leave it as an optional parameter for starters. Thoughts?

Reply:
Good catch, I intended to leave it as optional hence the default fallback to recaptcha but missed this here. Thank you